src/ProjectSettings.h

```diff
@@ -102,6 +102,10 @@ class AUDACITY_DLL_API ProjectSettings final
    void SetBrushRadius(int brushRadius) { mCurrentBrushRadius = brushRadius; }
    int GetBrushRadius() const { return mCurrentBrushRadius; }
 
+   // Keep track of hopping size of the brush (for time rounding when adding spectral data)
```

Comment:
Is this entire commit only for a debugging feature that you don't use in the final product?

Reply:
You are right, these lines should be removed from PR